packages/dashboard/src/components/Nav/NavLink.tsx

```diff
@@ -50,6 +49,9 @@ export const NavLink: React.SFC<INavLinkProps> = (props: INavLinkProps) => {
     display: rightIconName === 'OpenInNewWindow' ? 'none' : 'inline-block'
   };
 
+  // if there is a href and target honour href and do not execute onclick
+  const onClick = !!href && !!target ? undefined : props.onClick;
```

Comment:
For some scenarios, it's necessary to both execute the click handler and do the default navigation specified by the href. So this change isn't acceptable.

If you explain more about your scenario, maybe I can help come up with an alternative that will achieve your needs without breaking others.

Reply:
Actually never mind, I see now that this is your dashboard-specific nav, not the general Fabric nav. So this change is okay.

Reply:
Thanks. I reverted the change anyway.